src/main.ts

```diff
@@ -46,18 +46,19 @@ async function start () {
   const windowLocationPathname = window.location.pathname // eg, /business/CP1234567/...
   const windowLocationOrigin = window.location.origin // eg, http://localhost:8080
 
-  // first load the configuration, then set base route and check business id
-  const configurationStore = useConfigurationStore()
-  configurationStore.loadConfiguration()
-
-  setBaseRouteAndBusinessId(windowLocationPathname, processEnvBaseUrl, windowLocationOrigin) // may throw an error
-
   // initialize Launch Darkly
+  window['ldClientId'] = import.meta.env.VUE_APP_BUSINESS_FILING_LD_CLIENT_ID
   if (window['ldClientId']) {
     console.info('Initializing LaunchDarkly...') // eslint-disable-line no-console
     await InitLdClient()
   }
```

Comment:
I had to move the LD init before the config store, because the store needs the FF.